cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -101,6 +103,13 @@ def filtered_list(cls):
         if not XBlockStudioConfigurationFlag.is_enabled():
             filtered_list.append('allow_unsupported_xblocks')
 
+        # TODO: Alex Dusenbery 2017-06-22
+        # Before we roll out the auto-certs feature, move this to a good, shared
+        # place such that we're not repeating code found in LMS.
+        switches = WaffleSwitchNamespace(name=u'certificates', log_prefix=u'Certificates: ')
+        if not (switches.is_enabled(u'self_paced_only') or switches.is_enabled(u'instructor_paced_only')):
```

Comment:
Actually, this is not just a Nit.
Self-paced will be enabled well before Instructor-paced is.  So we'll want to hide this until instructor-paced is supported.